spec/features/welcome/user_sees_welcome_page_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe "shelters welcome", type: feature do
+  it "sees welcome message" do
+
+    visit "/"
+
+    expect(page).to have_content("Welcome to Adopt, Don't Shop!")
+  end
```

Comment:
Test failed because it did not recognize the method 'visit', so I added capybara to the rails_helper.rb.